scripts/create_pr.sh

```diff
@@ -13,30 +13,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-./vuln-list-k8s -vuln-list-dir "$VULN_LIST_DIR"
-
-cd "$VULN_LIST_DIR" || exit 1
+./cve-feed-osv
 
 if [[ -n $(git status --porcelain) ]]; then
   # List changed files
-  CHANGED_FILES=$(git ls-files . --exclude-standard --others | grep "CVE")
-  REPO="$REPOSITORY_OWNER/$VULN_LIST_DIR"
+  CHANGED_FILES=$(git ls-files . --exclude-standard --others --modified | grep "vulns/CVE")
+  REPO="$REPOSITORY_OWNER/cve-feed-osv"
```

Comment:
Do you think we don't need to set the default value so we can run this script locally?
e.g. bash: https://coderwall.com/p/s8n9qa/default-parameter-value-in-bash